packages/x-data-grid/src/hooks/features/listView/gridListViewSelectors.ts

```diff
@@ -1,6 +1,7 @@
 import { GridStateCommunity } from '../../../models/gridStateCommunity';
 
 /**
- * Get a list column definition
+ * Get the list column definition
+ * @category List View
  */
 export const gridListColumnSelector = (state: GridStateCommunity) => state.listViewColumn;
```

Comment:
For the `unstable_` features, we used to add `unstable_` prefix with the associated public methods (like selectors) too [[example](https://github.com/mui/mui-x/pull/11698)], what do you think about doing the same here?
```suggestion
export const unstable_gridListColumnSelector = (state: GridStateCommunity) => state.listViewColumn;
```

Reply:
We used to disable the rule for such lines.
No need to do it for the unexported selector 👍